### ISSUE TITLE
DALI-dataloader

### ISSUE THREAD

**Issue by Amin-Azar** [d-li14/mobilenetv2.pytorch#15]
I wonder if you have updated the DALI dataloader since last push? If not, I'll send a PR that makes fixed dali dataloader for newer versions of dali/nvidia stuff.

**Comment by d-li14** [comment]
It is not recommended now due to some known issues related to model accuracy.

**Comment by dingqingy** [comment]
Hi @d-li14,

Could you please provide a reference to the issues that you mentioned?

Thanks!

**Comment by d-li14** [comment]
I guess someone like this https://github.com/NVIDIA/DALI/issues/400.
